__tests__/app.test.js

```diff
@@ -101,3 +101,33 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("returns 200 status code and an array of articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(response.status).toBe(200);
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles.length).toBe(13);
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("comment_count");
+        });
```

Comment:
You can be a lot more specific here about what you expect from the article objects, what other properties would you expect it to contain ?